SpanishFilter/sections/specific.txt

```diff
@@ -6,6 +6,7 @@
 ! Incorrectly blocked domains can be excluded using regex(synchronize this regexp in all filters)
 /^(?!.*(bootstrapcdn.com|cdn.ampproject.org|cloudflare.com|disqus.com|disquscdn.com|dmca.com|ebacdn.com|facebook.net|fbcdn.net|fluidplayer.com|fontawesome.com|github.io|google.com|googleapis.com|googletagmanager.com|gstatic.com|jquery.com|jsdelivr.net|jwpcdn.com|jwplatform.com|polyfill.io|recaptcha.net|shrink.pe|twitter.com|ulogin.ru|unpkg.com|userapi.com|vidazoo.com|vk.com|yandex.|yastatic.net|ytimg.com|zencdn.net|player|youtube.com|cackle.me|googleoptimize.com|vuukle.com)).*$/$third-party,script,domain=thepiratefilmestorrent.biz|serieslatinoamerica.tv|pelispop.com
 !
+goblinsguide.com##.textwidget > p > a[href] > [width="270"][height="240"]
```

Comment:
Image is blocked by this rule `-300x250-$~xmlhttprequest`, so I think that it would be better to block images with `[src*="-300x250-"]`
For example something like:
`goblinsguide.com##a > img[src*="-300x250-"]`

Reply:
Ok. By the way, why the app just doesn't hide the banner? It's quite annoying in some cases and depends on one's mood.

Reply:
Banner is hidden on my end, but this image has `[width="270"]` and `[height="240"]` attributes which cause that there is ad placeholder with clickable link.
If I'm not wrong, extensions add `display: none !important;` (and some other styles) to blocked images/frames.

Reply:
By the way, it seems that it's not a Spanish website only, could you please move this rule to Base filter?

Reply:
Ok.